crates/bevy_picking/src/mesh_picking/ray_cast/intersections.rs

```diff
@@ -37,9 +37,7 @@ pub(super) fn ray_intersection_over_mesh(
     backface_culling: Backfaces,
 ) -> Option<RayMeshHit> {
     if mesh.primitive_topology() != PrimitiveTopology::TriangleList {
-        error!(
-            "Invalid intersection check: `TriangleList` is the only supported `PrimitiveTopology`"
-        );
+        // ray_mesh_intersection assumes vertices are laid out in a triangle list
         return None;
     }
 
```

Comment:
Comment on lines 47/53:

While I didn't encounter these, at least [one other person](https://github.com/bevyengine/bevy/issues/16065#issuecomment-2439420880) did.

* How would you feel about removing all of the error logging?
* For any that are deemed useful enough to remain, consider lowering to a lower log level (and at the very least switching to `*_once!`)

Reply:
Thanks for pointing this out!